recipes/mold/all/test_package/conanfile.py

```diff
@@ -0,0 +1,11 @@
+import os
+from conans import ConanFile, tools
+from conan.tools.build import cross_building
+
+class TestPackageConan(ConanFile):
+    settings = "os", "arch"
+
+    def test(self):
+        if not cross_building(self):
+            self.run("echo $LD", run_environment=True)
```

Comment:
Where does this `LD` variable come from? From a quick search it doesn't look like this is used by anything and mold's usage instructions also don't have this.

Reply:
CC, CXX and LD are standard variables for C compiler, C++ compiler and linker.